playwright/_impl/_browser.py

```diff
@@ -189,6 +189,17 @@ async def stop_tracing(self) -> bytes:
         encoded_binary = await self._channel.send("stopTracing")
         return base64.b64decode(encoded_binary)
 
+    async def __aenter__(self) -> "Browser":
```

Comment:
We never expose _impl to the users, so I don't think we should implement contexts for them.

Reply:
Indeed we never expose _impl to users, users does not get any object from _impl, they always it is wrapped